src/tests/functional/plugin/shared/include/behavior/ov_plugin/core_integration.hpp

```diff
@@ -994,6 +995,27 @@ TEST_P(OVClassLoadNetworkTest, LoadNetworkWithBigDeviceIDThrows) {
     }
 }
 
+TEST_P(OVClassLoadNetworkWithAutoBatchingTest, LoadNetworkWithAutoBatchingNoThrows) {
+    ov::Core ie = createCoreWithTemplate();
+    ov::CompiledModel model;
+
+    // Disable Auto Batching
+    OV_ASSERT_NO_THROW(model = ie.compile_model(actualNetwork, deviceName, {
+        ov::hint::performance_mode(ov::hint::PerformanceMode::THROUGHPUT),
+        ov::hint::allow_auto_batching(false)
+    }));
+    ASSERT_THROW(model.get_property(CONFIG_KEY(AUTO_BATCH_DEVICE_CONFIG)).as<std::string>(), ov::Exception);
```

Comment:
why do we use old config key here?  can we use `ov::device::priorities` ?

Reply:
This is the definition of `ov::device::priorities`: `static constexpr Priorities priorities{"MULTI_DEVICE_PRIORITIES"};`, which is not support by Auto Batching plugin yet.

Reply:
Add property `ov::device::batch_priorities` for this old key `AUTO_BATCH_DEVICE_CONFIG`.